File: csharp/HearthstoneReplays/ReplayData/Meta/Choice.cs
#region

using System.Xml.Serialization;

#endregion

namespace HearthstoneReplays.ReplayData.Meta
{
	[XmlRoot("Choice")]
	public class Choice : GameData
	{
		[XmlAttribute("entity")]
		public int Entity { get; set; }

		[XmlAttribute("index")]
		public int Index { get; set; }
	}
}

File: java/src/main/java/info/hearthsim/hsreplay/parser/replaydata/meta/Choice.java
package info.hearthsim.hsreplay.parser.replaydata.meta;

import info.hearthsim.hsreplay.parser.replaydata.GameData;

import javax.xml.bind.annotation.XmlAccessOrder;
import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorOrder;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlAttribute;
import javax.xml.bind.annotation.XmlRootElement;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@XmlRootElement(name = "Choice")
@XmlAccessorType(XmlAccessType.FIELD)
@XmlAccessorOrder(XmlAccessOrder.ALPHABETICAL)
@ToString
public class Choice extends GameData {

	@XmlAttribute(name = "entity")
	private int entity;

	@XmlAttribute(name = "index")
	private int index;

}
